apps/prairielearn/src/lib/cookie.ts

```diff
@@ -11,6 +11,17 @@ export function shouldSecureCookie(req: Request): boolean {
   return !config.devMode || req.protocol === 'https';
 }
 
+/**
+ * Helper function to clear a cookie regardless of if it was set with and
+ * explicit domain or not.
+ */
+export function clearCookie(res: Response, name: string): void {
```

Comment:
We could make this function "clever" so that if we give it `pl2_access_as_administrator`, it automatically clears `pl_access_as_asministrator`. This would reduce the amount of code we have to stick in every other file, at the cost of slightly obscuring what's happening.